app/forms/steps/claim_reason_form.rb

```diff
@@ -0,0 +1,48 @@
+require 'steps/base_form_object.rb'
+
+module Steps
+  class ClaimReasonForm < Steps::BaseFormObject
+    attribute :claim_reason, :value_object, source: ClaimReason
+
+    attribute :rep_order_date, :multiparam_date
+    attribute :cntp_order, :string
+    attribute :cntp_date, :multiparam_date
```

Comment:
we can removed the previous fields and validations are they can't be updated via this form